styleguide/dataset.html

```diff
@@ -0,0 +1,166 @@
+<!DOCTYPE html>
+<html lang="en" class="">
+<head>
+
+  <title>NHS - Replacement Publication System - Dataset</title>
+
+  <meta charset="UTF-8" />
+  <meta name="title" content="NHS - Replacement Publication System" />
+  <meta name="viewport" content="width=device-width, minimum-scale=1.0, maximum-scale=1.0" />
+  <link rel="stylesheet" href="../repository-data/webfiles/src/main/resources/site/css/demo.css" />
```

Comment:
This path appears to be pointing to a file that is dynamically generated from `demo.less` and, unless the build is run, the `*.css` version does not get generated leaving the templates broken.

How about generating the `demo.css` once, storing it somewhere under `styleguide` directory and changing this path to point to it? Once committed, this will make this file always be available to the HTML ones.

Reply:
I would like to avoid duplicated CSS - it opens the door to confusion, and people changing css in `/styleguide` folder directly :(